### PR TITLE
504 - move cib fields

### PR DESCRIPTION
**Pull request by marcellmueller**
https://app.zenhub.com/workspaces/ccbc---delivery-board-61c0c1204c8bcb001491c5f3/issues/bcgov/conn-ccbc-portal/504

### CODE REVIEW

File: app/tests/components/Review/Review.test.tsx [hunk 1]
```diff
@@ -236,28 +236,6 @@ describe('The Project funding  section', () => {
...
-
-    expect(
-      document.getElementById('infrastructureBankFunding2526')
-    ).toHaveTextContent(`2025-26`);
-
-    expect(
-      document.getElementById('totalInfrastructureBankFunding')
-    ).toHaveTextContent(
-      `Total amount requested under Canadian Infrastructure Bank`
-    );
```
Comment (matthieu-foucault) [inline app/tests/components/Review/Review.test.tsx:260]: We need a test to ensure that those now appear in the Other Funding Sources section

File: app/formSchema/pages/otherFundingSources.ts [hunk 2]
```diff
@@ -6,6 +6,26 @@ const otherFundingSources = {
       'Identify sources of funding that you expect to secure to cover all project costs, not including the Province of British Columbia, the Universal Broadband Fund or the Canadian Infrastructure Bank. Please only include loans that you anticipate receiving from a program or granting agency. Any other loans must be included in the applicant funding.',
```
Comment (matthieu-foucault) [inline app/formSchema/pages/otherFundingSources.ts:6]:
```suggestion
      'Identify sources of funding that you expect to secure to cover all project costs, not including the Province of British Columbia or the Universal Broadband Fund. Please only include loans that you anticipate receiving from a program or granting agency. Any other loans must be included in the applicant funding.',
```

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: app/tests/components/Review/Review.test.tsx (hunk 1 above)
Comment (matthieu-foucault) [reply app/tests/components/Review/Review.test.tsx:260]: Right, so we're waiting for the refactoring to be done to mark the issue complete then?

Comment (marcellmueller) [reply app/tests/components/Review/Review.test.tsx:260]: Yeah I guess it's up to you if you want me to spend time updating this. In a hypothetical situation where the review refactor didn't get completed I suppose we would want this to be updated.

**Pull request closed without merging**